### PR TITLE
feat: display MS org name logo [INTEG-1782]

### PR DESCRIPTION
**Pull request by whitelisab**
## Purpose

This PR updates the access section of the MS Teams app config page to display the organization name and logo after a user authenticates with Microsoft. 

Screenshot 2024-02-14 at 9 38 06 AM

## Approach

While the tenant Id and the username can be accessed from the MSAL hook, we need to call the MS Graph API in order to get the organization name and logo. I created a new `msGraphApi` util class to work with the API. This class handles getting the access token for the API request, calling the correct endpoints, and error handling. 

If there is an error getting the organization name and logo, these items won't display and a notification will show saying that there is error getting organization details

Screenshot 2024-02-14 at 8 45 38 AM

In order to create tests for `msGraphApi`, I utilized MSW to mock the HTTP requests. 

## Testing steps

Navigate to the MS Teams development space and authenticate to our sandbox account to see the org name and logo.

## Dependencies and/or References

See my notes in confluence for more information about working with the MS Graph API.

### CODE REVIEW

File: apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.styles.ts [hunk 1]
```diff
@@ -7,4 +7,8 @@ export const styles = {
     width: 16px;
   }
 `),
+  orgLogo: css({
+    /** Setting size to default size for favicon in MS organizational branding */
+    height: 32,
```
Comment (matthew-gordon) [inline apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.styles.ts:12]: this is more of a general question, but curious if it makes sense for us to start using tokens for sizing values? Can't remember where I saw that suggested recently, but what are your thoughts?

**Review by matthew-gordon — APPROVED** [review]
This looks good! Thank you for walking through it yesterday!

File: apps/microsoft-teams/frontend/src/utils/msGraphApi.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  readonly tenantId: string;
+
+  constructor(instance: IPublicClientApplication, account: AccountInfo) {
+    this.baseUrl = graphConfig.graphOrgEndpoint;
+    this.instance = instance;
+    this.account = account;
+    this.tenantId = account.tenantId;
+  }
+
+  /**
```
Comment (ethan-ozelius-contentful) [inline apps/microsoft-teams/frontend/src/utils/msGraphApi.ts:48]: love to see the jsdocs!

File: apps/microsoft-teams/frontend/src/utils/msGraphApi.ts [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private validateResponseStatus = (response: Response, responseJson: any) => {
+    if (response.status >= 400) {
+      const apiErrorResponse = {
+        status: response.status,
+        message: responseJson?.error?.message,
+      };
+
+      throw new MsGraphApiError(apiErrorResponse);
```
Comment (ethan-ozelius-contentful) [inline apps/microsoft-teams/frontend/src/utils/msGraphApi.ts:103]: Does this get caught anywhere?  If not, is there a risk of causing a runtime error that might make MS teams app unusable?

**Review by ethan-ozelius-contentful — APPROVED** [review]
Nice work!

File: apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.styles.ts (hunk 1 above)
Comment (whitelisab) [reply apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.styles.ts:12]: Yes, I've definitely used the [Forma36 tokens](https://f36.contentful.com/tokens/color-system), and we should try to use those whenever possible. They only have spacing values or sizing on specific Forma components, so I'm not sure it's the best use case here unless I'm missing something!

File: apps/microsoft-teams/frontend/src/utils/msGraphApi.ts (hunk 2 above)
Comment (whitelisab) [reply apps/microsoft-teams/frontend/src/utils/msGraphApi.ts:48]: Yes, it was a great suggestion from you for us to use this more!

**Pull request merged**